js/web/test/data/ops/slice.jsonc

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "name": "Slice float32",
+    "operator": "Slice",
+    "attributes": [],
+    "cases": [
+      {
+        "name": "T[5] T[1] T[1] (float32)",
+        "inputs": [
+          {
+            "data": [
+              0.3964604139328003, -0.8916832804679871, -1.6578896045684814, 1.960708737373352, 1.181204915046692
+            ],
+            "dims": [5],
+            "type": "float32"
+          },
+          { "data": [3], "dims": [1], "type": "int64" },
+          { "data": [4], "dims": [1], "type": "int64" }
+        ],
+        "outputs": [{ "data": [1.960708737373352], "dims": [1], "type": "float32" }]
+      }
+    ]
+  },
+  {
+    "name": "Slice int32",
+    "operator": "Slice",
+    "attributes": [],
+    "cases": [
+      {
+        "name": "T[5] T[1] T[1] (int32)",
+        "inputs": [
+          { "data": [0, 0, -1, 1, 0], "dims": [5], "type": "int32" },
+          { "data": [3], "dims": [1], "type": "int64" },
+          { "data": [4], "dims": [1], "type": "int64" }
+        ],
+        "outputs": [{ "data": [1], "dims": [1], "type": "int32" }]
+      }
+    ]
+  },
+  {
+    "name": "Slice bool",
```

Comment:
I think to support the `bool` type, you need to change the shader code in `slice.ts` since 4 bool values will be packed into one u32.  So it's better for this PR only add the int32 type. You can provide a separate PR to support the `bool` type.

Reply:
Done